tools/tox_tools/dose_responses/.shed.yml

```diff
@@ -0,0 +1,9 @@
+categories:
+  - Ecology
+  - Text Manipulation
+description: Toxicity prediction using QSAR models
+long_description: Toxicity prediction using QSAR models. Possible to select pre-defined models as well as user inputs.
+name: Dose Response Curve for Toxicological Risk Assessement
+owner: ufz
```

Comment:
That's a good point. We would need an toolshed user `ufz`.

Reply:
For background the user belonging to this api key https://github.com/Helmholtz-UFZ/galaxy-tools/blob/e409f80039e6d17577b5c52b97a56d915f876822/.github/workflows/pr.yaml#L413 needs write access to the tool repo in the tool shed. 

I think having an UFZ user (`ufz`) would be a good idea.